.github/workflows/ci.yml

```diff
@@ -56,6 +56,60 @@ jobs:
           java -jar target/dtls-fuzzer.jar args/ctinydtls/learn_ctinydtls_client_psk -roundLimit 5
           experiments/scripts/diff_hyps.sh experiments/models/regression/clients/rounds/ctinydtls_client_psk output/ctinydtls_client_psk 5
 
+  ETinyDTLS_Server_psk:
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y openjdk-8-jdk maven
+          sudo update-java-alternatives --set java-1.8.0-openjdk-amd64
+          java -version
+
+      - name: Install DTLS-Fuzzer
+        run: ./install.sh
+
+      - name: Setup SUT
+        run: ./setup_sut.sh etinydtls
+
+      - name: Test Eclipse TinyDTLS Server (psk)
+        run: |
+          java -jar target/dtls-fuzzer.jar args/etinydtls/learn_etinydtls_server_psk -roundLimit 5 -inputResponseTimeout CHANGE_CIPHER_SPEC:250
```

Comment:
Put the two `-roundLimit 5` _last_ on the two `java -jar ...` command`, for analogy with the other similar lines in this file and for maintaining the correspondence with the `5` in the line(s) below